beem/__init__.py

```diff
@@ -44,6 +45,18 @@ def print_publish_stats(stats):
     print("Total time            %.2f secs" % stats["time_total"])
 
 
+def json_dump_stats(stats, path):
+    """
+    write the stats object to disk.
+    """
+    try:
+       with open(path, 'w') as f:  
+           json.dump(stats, f)
+       print("Wrote stats to: {}".format(path))
+    except IOError:
+        print "Couldn't dump JSON stats to: {}".format(path)
```

Comment:
you should at least use the function form here so there's a chacne of this working with py3.  (and just to be consistent with other uses)